Corona-Warn-App/src/test/java/de/rki/coronawarnapp/ccl/dccwalletinfo/model/DccWalletInfoParserTest.kt

```diff
@@ -0,0 +1,135 @@
+                )
+            )
+        )
+    )
+
+    private val vaccinationState = VaccinationState(
+        visible = true,
+        titleText = SingleText(
+            type = "string",
+            localizedText = mapOf("de" to "Impfstatus"),
+            parameters = listOf()
+        ),
+        subtitleText = PluralText(
+            type = "plural",
+            quantity = 25,
+            localizedText = mapOf(
+                "de" to QuantityText(
+                    zero = "Letzte Impfung heute",
+                    one = "Letzte Impfung vor %u Tag",
+                    two = "Letzte Impfung vor %u Tagen",
+                    few = "Letzte Impfung vor %u Tagen",
+                    many = "Letzte Impfung vor %u Tagen",
+                    other = "Letzte Impfung vor %u Tagen"
+                )
+            ),
+            parameters = listOf(
+                Parameters(
+                    type = "date",
+                    value = "2021-12-01",
+                    format = "date-diff-now",
+                    unit = "day"
+                )
+            )
+        ),
+        longText = SingleText(
+            type = "string",
+            localizedText = mapOf("de" to "Sie haben nun alle derzeit [...]"),
+            parameters = listOf()
+        ),
+        faqAnchor = "dcc_admission_state"
+    )
+
+    private val admissionState = AdmissionState(
+        visible = true,
+        badgeText = SingleText(
+            type = "string",
+            localizedText = mapOf("de" to "2G+"),
+            parameters = listOf()
+        ),
+        titleText = SingleText(
+            type = "string",
+            localizedText = mapOf("de" to "Status-Nachweis"),
+            parameters = listOf()
+        ),
+        subtitleText = SingleText(
+            type = "string",
+            localizedText = mapOf("de" to "2G+ PCR-Test"),
+            parameters = listOf()
+        ),
+        longText = SingleText(
+            type = "string",
+            localizedText = mapOf("de" to "Ihre Zertifikate erfüllen [...]"),
+            parameters = listOf()
+        ),
+        faqAnchor = "dcc_admission_state"
+    )
+
+    private val dccWalletInfo = DccWalletInfo(
+        admissionState = admissionState,
+        vaccinationState = vaccinationState,
+        verification = verification,
+        boosterNotification = boosterNotification,
+        mostRelevantCertificate = mostRelevantCertificate,
+        validUntil = "2022-01-14T18:43:00Z"
+    )
+
+    @Test
+    fun `Serialize DCCWalletInfo`() {
```

Comment:
I think here we are testing the *de*serialization so I think the test name is wrong? If I am correct then also the function name below needs to be changed.

Reply:
done